packages/itwinui-react/src/core/Popover/Popover.tsx

```diff
@@ -193,8 +197,16 @@ export const usePopover = (options: PopoverOptions & PopoverInternalProps) => {
 // ----------------------------------------------------------------------------
 
 type PopoverPublicProps = {
+  /**
+   * Content displayed inside the popover.
+   */
   content?: React.ReactNode;
   children?: React.ReactNode;
+  /**
+   * Whether the popover adds recommended CSS for background-color, box-shadow, etc.
```

Comment:
Would be clearer to say *to what* this be added to

Reply:
added